test/verifiedTests/testSBML/testReadSBML.m

```diff
@@ -122,23 +102,12 @@
 
     model.lb = model.lb(:);
     model.ub = model.ub(:);
-
+    model.comps = {'e'};
+    model.compNames = {'ExtraCellular'};
     % add the fields outputted by readSBML
     [model.modelVersion.SBML_level, model.modelVersion.SBML_version, model.modelVersion.fbc_version] = deal(3, 1, 2);
```

Comment:
readSBML removes any empty fields. The only fields guaranteed to be present in a model returned by readSBML are (if there are any reactions/mets in the SBML file):
S
lb
ub
c
osense
csense
b
rxns
mets

Everything else could potenitally be removed.